crates/forge/src/lib.rs

```diff
@@ -105,6 +107,8 @@ pub async fn run(
     let not_filtered: usize = test_crates.iter().map(|tc| tc.test_cases.len()).sum();
     let filtered = all_tests - not_filtered;
 
+    verify_scarb_version_if_available_gas_used(&test_crates)?;
```

Comment:
It's too early to fail, we should only fail the specific test case imo (in `run_test_case`)

Reply:
Why? Let's assume someone has 100 tests with `available_gas` attribute. He will get 200 error messages with info about the scarb version

Reply:
Still there is no reason why the rest of the tests couldn't be run. Also that's how we did it before - we only failed the specific test when the attribute was specified, not everything